script/lookup-commit.sh

```diff
@@ -25,12 +25,17 @@ LORE_GIT_DIR="$(dirname "$0")/../.git/lore.kernel-git"
 test -n "$GITGIT_DIR" ||
 GITGIT_DIR="$(dirname "$0")/../.git/git-worktree"
 
+mailrepo="${GITGIT_MAIL_REPO:-dev.azure.com/gitgitgadget/git/_git/lore-git}"
+mailremote="${GITGIT_MAIL_REMOTE:-lore.kernel.org/git}"
+mailepoch="${GITGIT_MAIL_EPOCH:-1}"
```

Comment:
I have a slight preference to include the slash with the value.

And now that I notice it, I do have a preference for snake case in shell scripts: `mail_repo`, `mail_remote`, `mail_epoch` and `git_remote`.

Reply:
> I have a slight preference to include the slash with the value.

I presumed you meant `mail_epoch` to avoid a trailing slash if there is no epoch.